reference/abandoned_cart_emails.v3.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.0
 info:
-  title: Abandoned Cart Emails V3 API (open beta)
+  title: Abandoned Cart Emails V3 API
```

Comment:
```suggestion
  title: Abandoned Cart Emails
```

Reply:
going with the level of detail available in the sidebar & the titles of other endpoints.